docs/guides/admin/docs/configuration/firewall.md

```diff
@@ -0,0 +1,83 @@
+Firewall
+========
+
+Opencast relies on a lot of network communication and although not strictly necessary,
+it usually makes sense to configure a firewall for additional protection
+
+This describes a lot of the communication happening between servers in the most common use cases.
+In certain set-ups and with certain types of integrations, these may deviate slightly.
+
+General rules are:
+
+- Users communicate with Opencast via HTTP(S)
+- Capture agents communicate with Opencast via HTTP(S). All communication is established by the agent.
```

Comment:
This is the only item with periods.  The lines in the diagram are already directional, maybe omit the last sentence.  Or add periods to all of them :)